json2jx-doc.md

```diff
@@ -0,0 +1,889 @@
+# Converting JSON to JX #
```

Comment:
@trshaffer Please take a look. I perused this file, but this is more your domain.

Reply:
I think this looks good. It's helpful as an intro for workflow people who don't necessarily care about all the details of JX